js/job.jsx

```diff
@@ -476,23 +486,23 @@ function boxplot(data, node, tickFormat) {
 
 class Summarizer extends React.Component {
   render() {
-    var progress = this.props.progress;
-    var input_records = this.props.input_records;
-    var output_records = this.props.output_records;
-    var recordsPerSec = input_records != notAvailable ? Math.floor(input_records / (progress.totalTime / 1000)) : null;
+    var {progress} = this.props;
```

Comment:
Shouldn't all these `var`s be `let`s?

Reply:
yep, that's a future eslint fixup (i've done a bunch here but there are still 82 outstanding to fix)